src/IF.Lastfm.Core/IF.Lastfm.Core.csproj

```diff
@@ -3,8 +3,16 @@
   <PropertyGroup>
     <Version />
     <PackageVersion />
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Could this entry be merged with the ItemGroup on L20? Sure it doesn't matter, just curious.